Makefile

```diff
@@ -198,12 +200,9 @@ tesseract.built: tesseract-$(TESSERACT_VERSION)
 	cd $< && \
 		sh autogen.sh && \
 		PKG_CONFIG_PATH="$(LOCAL)/lib/pkgconfig" \
-		LEPTONICA_CFLAGS="-I$(LOCAL)/include/leptonica" \
 			./configure --prefix=$(LOCAL) && \
 		LDFLAGS="-L$(LOCAL)/lib"\
-			make -j$(CORES) && \
-		make install && \
-		make -j$(CORES) training-install && \
+			make -j$(CORES) install training-install && \
```

Comment:
I'm pretty sure there was a reason to have the `install` target separate from the default target. But cannot remember off the top of my hat. @wrznr?

Reply:
I'm pretty sure that the new code here is fine. :-) If it did not work, I'd consider that a bug in the Tesseract build system and certainly fix it, but I am not aware of such problems and use that kind of build very often.

Separating binary build and install is reasonable when installation needs root rights (sudo), but that's not the case here.

Reply:
> reasonable when installation needs root rights (sudo)

That was probably the reason.